Gulpfile.js

```diff
@@ -1,28 +1,17 @@
-var _ = require('underscore');
-var assets  = require('postcss-assets');
-var autoprefixer = require('autoprefixer-core');
-var deploy = require('gulp-gh-pages');
-var es = require('event-stream');
 var gulp = require('gulp');
-var karma = require('gulp-karma');
-var livereload = require('gulp-livereload');
-var ngAnnotate = require('gulp-ng-annotate');
-var postcss = require('gulp-postcss');
-var protractor = require('gulp-protractor').protractor;
-var rjs = require('gulp-requirejs');
-var sass = require('gulp-sass');
-var spawn = require('child_process').spawn;
-var uglify = require('gulp-uglify');
+var plumber = require('gulp-plumber');
 var webdriver = require('gulp-protractor').webdriver_standalone;
 
 var handleError = function (err) {
   console.log(err.name, ' in ', err.plugin, ': ', err.message);
   console.log(err.getStack());
-  process.exit(1);
+  this.emit('end');
```

Comment:
I know it kills watch but emit does not work well with CI - build will be marked as green even if karma-ci founds errors. In this thread possible solutions are discussed: https://github.com/gulpjs/gulp/issues/259